frontend/projects/ui/src/app/pages/server-routes/server-show/server-show.page.ts

```diff
@@ -307,7 +307,7 @@ export class ServerShowPage {
 
   async launchHttps() {
```

Comment:
When is this one invoked?

Reply:
<img width="1027" alt="Screenshot 2023-11-01 at 8 58 58 PM" src="https://github.com/Start9Labs/start-os/assets/9935159/e97a1992-3b79-4e4c-8b7d-1c21be8fbf5a">

Reply:
But specifically for tor?